src/incrementer.js

```diff
@@ -201,23 +198,37 @@ var Incrementer = {
      * @param {jQueryElement} $elt - plugin's element
      * @fires Incrementer#plus.incrementer
      */
-    _inc: function($elt) {
+    _inc: function _inc($elt) {
         var options = $elt.data(dataNs),
             current = parseFloat($elt.val() || 0),
             value;
-
         value = gamp.add(current, options.step);
+
+        if ($elt[0].getAttribute('name') === 'normalMaximum') {
+            options.max = parseFloat($elt.parent().parent().find("[name='normalMaximum']").attr('value')) + 1;
+            options.min = parseFloat($elt.parent().parent().find("[name='normalMinimum']").attr('value'));
+        } else {
+            options.max = parseFloat($elt.parent().parent().find("[name='normalMaximum']").attr('value'));
+            options.min = parseFloat($elt.parent().parent().find("[name='normalMinimum']").attr('value')) + 1;
```

Comment:
The code `parseFloat($elt.parent().parent().find("[name='normalMinimum']").attr('value'))` is present in multiple places, do you think it's better to move to some getter function for better readability and to not introduce errors in future refactorings / changes?
Same for normalMaximum

Reply:
Done